src/test/java/org/ambraproject/wombat/controller/StaticResourceControllerTest.java

```diff
@@ -157,17 +173,119 @@ public String describeSource() {
 
   @Configuration
   @EnableWebMvc
-  @Import(WombatControllerTestConfig.class)
```

Comment:
I got the impression that the WombatControllerTestConfig contained a bunch of generally applicable test setup, but if it was only being used in one place I guess it has limited applicability and it makes sense to inline it.

Reply:
Yeah, I figure it's better to put that in the `ControllerTestConfig` that is automatically imported when you extend `ControllerTest`.